src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -10124,10 +10133,14 @@ void SetAverageAirFlow(EnergyPlusData &state,
 
     if (state.dataHVACVarRefFlow->VRFTU(VRFTUNum).OpMode == DataHVACGlobals::CycFanCycCoil &&
         state.dataHVACVarRefFlow->VRFTU(VRFTUNum).SpeedNum == 0) {
+        Real64 partLoadRat = PartLoadRatio;
+        if (partLoadRat == 0.0 && state.dataHVACVarRefFlow->VRFTU(VRFTUNum).SuppHeatPartLoadRatio > 0.0) {
+            partLoadRat = state.dataHVACVarRefFlow->VRFTU(VRFTUNum).SuppHeatPartLoadRatio;
+        }
```

Comment:
When cycling fan mode is used the PartLoadRatio = 0 while the supplemental heating coil PLR > 0. This change allows proper air flow over the supplemental heating coil.

Reply:
OK, so a tighter scoped partLoadRat variable.  I understand the intent, though part of me thinks it is worth renaming the variables so that someone can understand the difference without a deep dive...not necessary here if no other changes.